Google Scholar.js

```diff
@@ -887,11 +907,43 @@ ItemFactory.prototype.getAttachments = function (doctype) {
 	attachments = [];
 	for (i = 0, ilen = this.attachmentLinks.length; i < ilen; i += 1) {
 		if (!this.attachmentLinks[i]) continue;
-		attachments.push({
-			title:"Google Scholar Linked " + doctype,
-			url:this.attachmentLinks[i],
-			type:"text/html"
-		});
+		if (Zotero.Utilities.setMultiField) {
+			if ("string" === typeof this.attachmentLinks[i]) {
+				// URL (string)
+				if (this.attachmentLinks[i].slice(0, 1) === "/") {
```

Comment:
Still learning JavaScript ... Good catch on the (lack of) need for this. Where the attachment is a string, the code should just fall over to the existing Zotero-orthodox URL attachment block below. Will revise.